js/index.js

```diff
@@ -1,13 +1,200 @@
+    livelink:
+      'https://tadessejemal1.github.io/portfolio-mobile-work-and-about/',
+    sourcelink:
+      'https://github.com/Tadessejemal1/portfolio-mobile-work-and-about',
+  },
+  {
+    title: 'Project 5',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    featuredImage: './images/popup.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    livelink:
+      'https://tadessejemal1.github.io/portfolio-mobile-work-and-about/',
+    sourcelink:
+      'https://github.com/Tadessejemal1/portfolio-mobile-work-and-about',
+  },
+  {
+    title: "Post Project",
+    technologies: ['HTML', 'CSS', 'Java'],
+    featuredImage: './images/popup.png',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    livelink:
+      'https://tadessejemal1.github.io/portfolio-mobile-work-and-about/',
+    sourcelink:
+      'https://github.com/Tadessejemal1/portfolio-mobile-work-and-about',
+  },
+];
+let title;
+let medias;
+let img;
+let description;
+let buttons;
+let seeLiveButton;
+let sourceButton;
+for (let i = 0; i < projectSee.length; i++) {
+  projectSee[i].onclick = function createModal() {
+    popupModal.style.display = 'block';
+
+    title = document.createElement('h3');
+    title.setAttribute('class', 'modal-title');
+    title.appendChild(document.createTextNode(projects[i].title));
+
+    medias = document.createElement('div');
+    medias.setAttribute('class', 'modal-media');
+    for (let i = 0; i < projects[i].technologies.length; i++) {
+      const techDisplay = document.createElement('span');
+      techDisplay.setAttribute('class', 'badge');
+      techDisplay.appendChild(
+        document.createTextNode(projects[i].technologies[i])
+      );
+      medias.appendChild(techDisplay);
+    }
+    // projects[i].technologies.forEach((i) => {
+    //     const techDisplay = document.createElement('span');
+    //     techDisplay.setAttribute('class', 'badge');
+    //     techDisplay.appendChild(document.createTextNode(projects[i].technologies));
+    //     medias.appendChild(techDisplay);
+    // });
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: